IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -104,6 +104,7 @@ - (void)logWithLevel:(MSIDLogLevel)level
          containsPII:(BOOL)containsPII
             filename:(NSString *)filename
           lineNumber:(NSUInteger)lineNumber
+            function:(__unused NSString *)function
```

Comment:
should function be behind self.SourceLineLoggingEnabled switch as well?

Reply:
The __func__ strings can be a bit long, so I wasn't sure if you'd want them. Added.